apis/Google.Cloud.Asset.V1/docs/history.md

```diff
@@ -1,5 +1,19 @@
 # Version history
 
+# Version 2.3.0, released 2020-08-25
+
+- [Commit 8ef5dc2](https://github.com/googleapis/google-cloud-dotnet/commit/8ef5dc2): feat: add AnalyzeIamPolicy and ExportIamPolicyAnalysis RPCs
+- [Commit e7b75e6](https://github.com/googleapis/google-cloud-dotnet/commit/e7b75e6):
+  - feat: added support OutputResult in ExportAssetsResponse.
+  - Clients receive detailed OutputResult in ExportAssetsResponse.
+  - docs: updated existing docs.  
+- [Commit 6bde7a3](https://github.com/googleapis/google-cloud-dotnet/commit/6bde7a3):
```

Comment:
I'd probably unwrap this to just "Adds service comments in client documentation" in the same line as the commit.

Reply:
Done